### ISSUE TITLE
Won't get my Ether back

### ISSUE THREAD

**Issue by pauliax** [smithclay/etherimp#1]
I don't know if this was intended or not, but now I don't see a point in sending Ether to your contract because of this line:
previousOwner.transfer(msg.value);

Basically, this transfers new lowest price to the previous owner. So, imagine that I sent 0.03 Ether to your contract and became a king. Then you sent 0.01 Ether and stole my crown. I will be refunded with 0.01 Ether, so my 0.02 Ether is lost and can only be retrieved by the owner of the contract (you) by calling close() function. I think that's unfair and you should fix it.

**Comment by smithclay** [comment]
Agreed that it's unfair (and it wasn't intentional). Needs to be fixed.
